opentelemetry-sdk/benches/batch_log_processor.rs

```diff
@@ -0,0 +1,138 @@
+use criterion::{criterion_group, criterion_main, Criterion};
```

Comment:
please share the bench results for easy reference!

Reply:
Have added it. This mayn't be highly accurate, I believe slightly over-reported. But for now, we have baseline to compare. And can be improved over.